src/routers/booleansRouter.js

```diff
@@ -0,0 +1,12 @@
+const booleansController = require('./../controllers/booleansControllers');
```

Comment:
You can deconstruct your functions here too, but if it's clearer for you to have the controller itself on lines 7-10 that's also cool